utils.py

```
@@ -7,6 +7,10 @@
 from slugify import slugify
 import string
 import secrets
+import requests
+from os import getenv
```

Ruff (0.12.2)

`os.getenv` imported but unused

Remove unused import: `os.getenv`

(F401)